com.avaloq.tools.ddk.xtext/src/com/avaloq/tools/ddk/xtext/resource/ResourceDescriptionDelta.java

```diff
@@ -36,7 +35,7 @@
  * A resource description delta, with a "hasChanges" operation optimized for fingerprints. Additionally the new resource description is only soft referenced.
  */
 @SuppressWarnings({"PMD.CompareObjectsWithEquals", "PMD.NPathComplexity"})
-public class ResourceDescriptionDelta extends AbstractResourceDescriptionDelta {
+public final class ResourceDescriptionDelta extends AbstractResourceDescriptionDelta {

```

Comment:
Languages may and actually do customize it. It's up to the language-specific *ResoruceDescriptionManager.